src/com/theironyard/novauc/SpaceCraft.java

```diff
@@ -0,0 +1,53 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by Merlin on 2/7/17.
+ */
+public class SpaceCraft {
+    String country;
+    String name;
+    int FastestSpeed;
+    int FuelMaximumCapacity;
+    boolean isOperational;
+
+    public String getCountry() {
+        return country; }
+    public void setCountry(String country) {
+        this.country = country; }
+
+    public String getName() {
+        return name; }
+    public void setName(String name) {
+        this.name = name; }
+
+    public int getFastestSpeed() {
+        return FastestSpeed; }
+    public void setFastestSpeed(int fastestSpeed) {
+
+        if (FastestSpeed < 26,000 && FastestSpeed > 35,000){
+            this.FastestSpeed = fastestSpeed;
+        }
+        else {
+            System.out.println("Not a valid speed");
+        }
```

Comment:
Ross good job with the constructors, getters and setters.  Take a look at line 27 the format is correct but I think your greater than less than should be flipped.  The assignment also requires an if statement in ALL five classes.  Please write an if statement in a setter method for the other four classes and resubmit the assignment.  Let me know if you have any questions.